src/hardware/setup.rs

```diff
@@ -289,8 +289,15 @@ pub fn setup(
         mac
     };
 
+    let mut flash = {
+        let flash = stm32f4xx_hal::flash::LockedFlash::new(device.FLASH);
+        const SECTOR_SIZE: usize = 128 * 1024;
+        Flash::new(flash, 6 * SECTOR_SIZE)
```

Comment:
I don't get the change from `7*` to `6*` and the doubling of the reserved space size in the linker script. Could you explain?

Reply:
Ah. Can we write this as `(8 - 2)` instead of `6`? That was my confusion.

Reply:
FIxed in 7631955a768b033582db8a4a86747d3d2d792d5e